docs/tutorials/5_Event_Planning.rst

```diff
@@ -0,0 +1,73 @@
+Tutorial Five: Event Planning
+=============================
+
+Event's are integral to Vivarium but up to this point we've only talked about a couple of standard events emitted by the framework itself. We can emit our own events and that's key to some kinds of complex models. In this tutorial we'll talk about emitting and responding to custom events.
```

Comment:
No don't!  We want to bury the event system very far away from the modelers.  And eventually get rid of it completely.  Users are not coming from a web development or standalone application development perspective and event driven programming introduces really unnecessary complexity.